backend/app/api/endpoints/adapter/tasks.py

```
@@ -222,6 +225,41 @@ def search_tasks_by_title(
     return {"total": total, "items": items}
 
 
+@router.get("/archived", response_model=ArchivedTaskListResponse)
+def get_archived_tasks(
+    page: int = Query(1, ge=1, description="Page number"),
+    limit: int = Query(100, ge=1, le=200, description="Items per page"),
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.15.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)